app/src/styles.sass

```diff
@@ -164,17 +163,37 @@ $itero-mat-theme: mat-light-theme(( color: ( primary: $itero-mat-primary, accent
   right: 0
   display: flex
   flex-flow: row nowrap
-  width: 80%
+  overflow: hidden
+  transition: width 2s
+  min-width: 32px
+  max-width: 80vw
+  width: 80vw
   @media screen and (min-width: $breakpoint-tablet) and (max-width: $breakpoint-laptop)
-    width: 50%
+    max-width: 50vw
+    width: 50vw
   @media screen and (min-width: $breakpoint-laptop)
     display: none
-  .info-panel-icon #help-button
-    margin: 0.7em $small-space
+  .info-panel-content
+    overflow: hidden
   .info-panel-content .information
-    display: contents
+    margin: 0
+    padding: 0
+    border: none
+    width: 70vw
```

Comment:
This is not very accurate because 10vw is almost never equal to 32px. With screen width of 360px, there is a right margin of 4px, with a screen width of 499px, there is a right margin of 18px, and with a screen of 500px, the right margin is of -7px, which means that some part of the text may be hidden (try with `text-align: justify` to see it clearly).

My suggestions:
 - Use [calc](https://developer.mozilla.org/en-US/docs/Web/CSS/calc()) to compute precisely the width of .information.
 - Use [local variables](https://sass-lang.com/documentation/variables#scope) to store the width that appear more than once. Such a local variable is defined at line 70 of navtitle.component.sass and used on the following line, to give you an example.